SoftServeCarMobile/src/activity/journey/journey-activity/segment-control-activities/JourneyPage.tsx

```diff
@@ -1,11 +1,90 @@
-import React from 'react'
-import { Text, View } from 'react-native'
+import React, { useRef, useState } from 'react';
+import {
+    Text,
+    View,
+    StyleSheet
+} from 'react-native';
+import BottomSheet from 'reanimated-bottom-sheet';
+import MenuButton from '../../../../components/bottom-popup/MenuButton';
+import BottomPopup from '../../../../components/bottom-popup/BottomPopup';
 
-const JourneyPage = () => {
-    return (
-        <View>
-            <Text>Journey</Text>
+const JourneyPage = (props: any) => {
+
+    const myRef = React.useRef<BottomSheet>(null);
+
+    const renderInner = () => (
+        <View style={styles.panel}>
+            <MenuButton text="View profile"></MenuButton>
+            <MenuButton text="Message"></MenuButton>
+        </View>
+    )
+
+    if (props.isOpen == true) {
```

Comment:
please simplify this code: init snap to value in the variable and use ternary operator for it, ex:
let index = props.isOpen ? 1 : 0;
myRef?.current?.snapTo(index )